src/style.css

```diff
@@ -1,7 +1,70 @@
+*,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-  background-color: bisque;
+  background-color: white;
+}
+
+h1 {
+  padding: 10px;
+  text-align: center;
+}
+
+.flex {
+  display: flex;
+}
+
+.main {
+  align-items: center;
+  justify-content: space-around;
+  margin: 120px 170px;
+}
+
+#board {
+  list-style-type: none;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid black;
+}
+
+#board li,
+.input {
+  padding: 10px;
+}
+
+#board li:nth-child(even) {
+  background-color: gray;
+}
+
+form {
+  flex-direction: column;
+  text-align: center;
+}
+
+.scores {
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card {
+  width: 45%;
+}
```

Comment:
- [ ] I think it is better if you keep the horizontal alignment of your card class components to stick with the original wireframe design to fulfill the minimum styling. 
![image](https://user-images.githubusercontent.com/64894927/131400921-faa6d378-9e42-4fe3-b870-ce40bf885718.png)
![image](https://user-images.githubusercontent.com/64894927/131400978-15202d81-29e4-4fae-b716-edbf95454686.png)

Reply:
This issue is not properly resolved.

Reply:
Well addressed!! :clap: